eng/BuildTask.targets

```diff
@@ -6,6 +6,8 @@
   <PropertyGroup>
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <IsPackable>true</IsPackable>
+    <!-- Build Tasks should not include any dependencies -->
+    <SuppressDependenciesWhenPacking Condition="'$(SuppressDependenciesWhenPacking)' == ''">true</SuppressDependenciesWhenPacking>
```

Comment:
@mmitche @riarenas What are the odds of this breaking darc.exe?  I couldn't get my SDK to build without making this change, and this mirrors what arcades copy of this file looks like.  I don't like changing this file, but I don't have a good other way to fix this... should I set it to true manually in darc? Is there some way I can prove whatever I'm packing is valid?  I don't exactly know how to validate the packed version of darc.exe, and I have a pretty strong suspicion the build isn't going to validate that for me.

Reply:
I'm not sure.

You can build -> pack the repo -> install the global tool (like darc-init.ps1 does) and test it out.

Reply:
Could always install your locally built darc and make sure it works.

`build.cmd -pack` then edit https://github.com/dotnet/arcade-services/blob/04908ec285c0791886a1ed4509c8a8dd96a788b1/eng/common/darc-init.ps1#L27 to point to your local `artifacts/packages/debug/shipping/` directory, and pass the `--darcVersion` parameter as your locally produced version.

Reply:
Interesting... I think I also flipped the "shipping" bit.  Since I didn't think anything from the services repo got "shipped". but maybe darc does?  I'll see if I can flip darc.exe back to shipping and leave the rest non-shipping, since all the other servicey stuff we publish shouldn't get shipped anywhere.

Reply:
It seems to work in it's current format... is the "shipping" bit important?

Reply:
They're really all non-shipping because they dont' go to nuget.org

Reply:
Sweet. I'll leave it as-is then.  Everything seems to be working now, so I'm going to merge.